### ISSUE TITLE
custom acf blocks image url not showing

### ISSUE THREAD

**Issue by sunilsingh2019** [pristas-peter/wp-graphql-gutenberg#173]
I have created custom acf blocks other fields render fine but image field rendering values number find image for example

attributes: data: button: {title: 'DOWNLOAD RESUME', url: 'http://example.com.au/app/uploads/2022/12/SUNIL-SINGH2022.pdf', target: ''}enable_disable_module: "1"heading_1: "SUNIL "heading_2: "SINGH"hero_image: 17sub_heading: "I AM A FULL STACK WEB DEVELOPER"_button: "field_6345f391a8325"_enable_disable_module: "field_6346010b6dd8f"_heading_1: "field_6345ef9305807"_heading_2: "field_6345f52cfbe6e"_hero_image: "field_6345f1ce05809"_sub_heading: "field_6345f1a105808"[[Prototype]]: Objectmode: "edit"name: "acf/hero-module"

![image](https://user-images.githubusercontent.com/26164995/205461850-f90c452b-7d4a-49cb-aaaf-71c7ccdbb5b2.png)

**Comment by kevinpoot** [comment]
Yes, it seems the api will only return the id of the image instead of other options like url or array. Can we get a fix for this?
